src/components/biggive-search/biggive-search.scss

```diff
@@ -0,0 +1,120 @@
+.container {
+  @include standard-font();
+  @include all-round-shadow();
+  width: 400px;
+  padding: 1.5rem;
+
+  h2 {
+    @include standard-font();
```

Comment:
Do we need this on every child element?

Reply:
@NoelLH Yes, we need the same font for the search component's title all the time. It shouldn't ever change. Not sure what you mean by 'every child element' of a `h2`?

Reply:
@alihejazi I meant every child element of the parent `.container`, where you have already included the font.

Reply:
@NoelLH Good catch, it wasn't needed in the `h2` as you pointed out but I still needed it for the `input` field and for the `button`.